src/System.Private.Xml/tests/Writers/XmlWriterApi/CommonTests.cs

```diff
@@ -9,8 +9,6 @@
 using System.Text;
 using XmlCoreTest.Common;
 
-[assembly: System.Diagnostics.CodeAnalysis.ExcludeFromCodeCoverage]
```

Comment:
This `System.Diagnostics.CodeAnalysis.ExcludeFromCodeCoverage` type was previously included from the "Common" directory, but now that I've upgraded this project to `netstandard1.7`, that conflicts with the type `ExcludedFromCodeCoverage` from "System.Diagnostics.CodeAnalysis.dll", which no longer can be applied to assemblies. That's the reason I removed this attribute here.